docker/README.md

```diff
@@ -2,68 +2,58 @@
+Install [Docker for Desktop](https://www.docker.com/products/docker-desktop) on your local machine and _start Docker_ before continuing, as it will be needed to continue with the deployment process.
 
-## Create your DNS record
+For macOS users who use Homebrew, ensure you're using the _cask_ app-based version of Docker, not the default CLI version.
 
-The 1Password SCIM bridge requires SSL/TLS in order to communicate with your IdP. In order to use TLS, you must create a DNS record that points to your Docker node. _Do not attempt to perform a provisioning sync before the DNS records have been propogated_. The DNS record must exist and the SCIM bridge server must be running if you wish to have LetsEncrypt automatically issue a TLS certificate for your SCIM bridge. _Please refer to your cloud provider on how to setup a DNS record if you do not have one set up already or are experiencing difficulties doing so._
 
-## Prepare your 1Password Account
+## Docker Compose vs Docker Swarm
 
-Log in to your 1Password account [using this link](https://start.1password.com/settings/provisioning/setup).  It will take you to the setup page for the SCIM bridge.
+Using Docker, you have two different deployment options: `docker-compose` and Docker Swarm.
 
-Follow the on-screen instructions which will guide you through the following steps:
+Docker Swarm is the recommended option. _Please refer to your cloud provider on how to setup a remote Docker Swarm Cluster if you do not have one set up already or are experiencing difficulties doing so._
 
-* Create a Provision Managers group
-* Create and confirm a Provision Manager user
-* Generate your SCIM bridge credentials
+`docker-compose` is very useful for testing, but it is not recommended for use in a production environment. The `scimsession` file is passed into the docker container via an environment variable, which is less secure than Docker Swarm secrets or Kubernetes secrets, both of which are supported, and recommended.
 
-You can then download the `scimsession` file and save your bearer token.  The `scimsession` file contains the credentials for the new Provision Manager user.  This user will create, confirm, and suspend users, and create and manage access to groups.  You should use an email address that is unique.
 
-The bearer token and scimsession file combined can be used to sign in to your Provision Manager account. You’ll need to share the bearer token with your identity provider, but it’s important to **never share it with anyone else**. And never share your scimsession file with **anyone at all**.
-
-You should move your newly created `scimsession` file into the `scim-examples` folder (the root folder of this repository which you cloned earlier).
+## Deploy SCIM bridge
 
+Connect to your remote Docker host from your local machine using the following:
 
-## Deploy SCIM bridge
+```
+eval %{docker-machine env dev}
```

Comment:
>eval %{docker-machine env dev}

Unless I'm missing something, this should be `eval $(docker-machine env $machine_name)`

Additionally the setup is not this simple. There is the docker-machine connection that has to be established and then keys stored etc. I would add a reference to the official linking docs (https://docs.docker.com/machine/reference/create/), the list of available drivers (https://docs.docker.com/machine/drivers/) and the option to use an existing vm.

EG: `docker-machine create --driver google --google-use-existing --google-project "My Project" "1P-SCIM-VM"`